### ISSUE TITLE
Executing "npx gulp lib" returns "Did you forget to signal async completion?" message

### ISSUE THREAD

**Issue by sanak** [Viglino/ol-ext#619]
From https://github.com/Viglino/ol-ext/issues/617#issuecomment-864508270.
```bash
$ npx gulp lib
[15:32:02] Using gulpfile ~/Projects/gtt-project/git/ol-ext/gulpfile.js
[15:32:02] Starting 'lib'...
[15:32:02] The following tasks did not complete: lib
[15:32:02] Did you forget to signal async completion?
```

Probably, one of the following answer fix will be necessary.
https://stackoverflow.com/questions/36897877/gulp-error-the-following-tasks-did-not-complete-did-you-forget-to-signal-async

**Comment by Viglino** [comment]
The lib script is no more used and was using an old version of gulp and didn't return a completion signal.
The last commit should solve the issue.

**Comment by sanak** [comment]
@Viglino Okay, thanks for quick fix, and sorry for wrong link in #621 PR... 🙇‍♂️ 
I confirmed that this issue is fixed on the commit https://github.com/Viglino/ol-ext/commit/abc94892e818c466562a66fe16601d72d3395466, so I will close this.
